translations/bnpchs-final-2018-World-Class-Lift.html

```diff
@@ -0,0 +1,71 @@
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  World Class University has a sky-high building for student residence. 
+  This residence is called "World Square" or often abbreviated as WS. 
+  WS building had a World Class Elevator. 
+  Because of the extraordinary quality, WS only needs one elevator to accommodate all of the collegers.
+</p>
+
+<p style="text-align: justify">
+  There are $$$N$$$ passenger in first floor inside the elevator right now.
+  The $$$i$$$-th passenger of the the elevator wants to get to $$$X_{i}$$$-th floor and has an elevator tolerance value of $$$Y_{i}$$$. 
+  The elevator tolerance value concludes the range of floors so that 
+  the $$$i$$$-th passenger won't get annoyed if the $$$i$$$-th passenger got dropped 
+  between floor $$$X_{i} - Y_{i}$$$ and $$$X_{i} + Y_{i}$$$ (inclusive). 
+  After the ride passenger can go to the desired floor using the stairs. 
+  Mind that every passenger could have a different value of elevator tolerance. 
```

Comment:
These do not match the original Bahasa Indonesia description

you only need to describe the definition of Elevator Tolerance Value in this paragraph